CorsixTH/Lua/utility.lua

```diff
@@ -391,12 +391,13 @@ end
 --!param options Option settings, table, 'detect_cycles' field boolean that
 --  ends recursion on a cycle, and 'max_depth' integer that ends recursion at the
 --  specified depth. By default initialized with "{detect_cycles = True}"
+-- 'start_at_one' field boolean that starts the count in serialize_string at 1 not 0.
```

Comment:
- The documentation is wrong. The entire point of function documentation is to explain what effect parameter values have, *without* having to read the code of the function. This documentation however doesn't explain anything, it just says "see code for understanding what it does".

- The proposed `start_at_one` addition excludes by design any other value as start level. Why? What makes `1` the one and only value that we ever need?

Reply:
https://www.lua.org/manual/5.3/manual.html#:~:text=define%20an%20opening-,long%20bracket,-of%20level%20n suggests the field name long_bracket_level_start or initial count.

Reply:
The page is more about the precise definition of such brackets (it's almost math). The reason they do that is because a few lines further they start discussing "brackets of another level", which doesn't make sense unless you first explain the level of a single long bracket.

So in my view that page is about the formal definition of Lua concepts. (What is a long bracket, how do you decide the level of such a bracket, what happens if you have several such brackets?)   
While i don't think it suggests exactly that name, but if you like the name, I am fine with it.